.github/workflows/check-config.yml

```diff
@@ -12,5 +12,8 @@ jobs:
       targets: ${{ steps.set-matrix.outputs.targets }}
     steps:
       - uses: actions/checkout@v2
-      - name: check if config.json is sorted alphabetically
-        run: .github/workflows/check-config.sh config.json
+      - name: check if configs/*.json is sorted alphabetically
+        run: |
+          for config in configs/*.json; do
+            .github/workflows/check-config.sh $config
```

Comment:
Output the file name here, or do it the fancy way by using a [group](https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#grouping-log-lines).

Reply:
I did use group, thanks for pointing me to it :)